src/Symfony/Component/Notifier/Bridge/Clickatell/Tests/ClickatellTransportTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Symfony\Component\Notifier\Bridge\Clickatell\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\HttpClient\MockHttpClient;
+use Symfony\Component\Notifier\Bridge\Clickatell\ClickatellTransport;
+use Symfony\Component\Notifier\Exception\LogicException;
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+use Symfony\Contracts\HttpClient\ResponseInterface;
+
+class ClickatellTransportTest extends TestCase
+{
+    public function testToString(): void
+    {
+        $transport = new ClickatellTransport('authToken', 'fromValue', $this->createMock(HttpClientInterface::class));
+        $transport->setHost('clickatellHost');
+
+        self::assertSame('clickatell://clickatellHost?from=fromValue', (string) $transport);
+    }
+
+    public function testSupports(): void
+    {
+        $transport = new ClickatellTransport('authToken', 'fromValue', $this->createMock(HttpClientInterface::class));
+
+        self::assertTrue($transport->supports(new SmsMessage('+33612345678', 'testSmsMessage')));
+        self::assertFalse($transport->supports($this->createMock(MessageInterface::class)));
+    }
+
+    public function testExceptionIsThrownWhenNonMessageIsSend(): void
+    {
+        $this->expectException(LogicException::class);
+
+        $transport = new ClickatellTransport('authToken', 'fromValue', $this->createMock(HttpClientInterface::class));
+        $transport->send($this->createMock(MessageInterface::class));
+    }
+
+    public function testExceptionIsThrownWhenHttpSendFailed(): void
+    {
+        $this->expectException(TransportException::class);
+        $this->expectExceptionMessage('Unable to send SMS with Clickatell: Error code 105 with message "Invalid Account Reference EX0000000" (https://documentation-page).');
+
+        $response = $this->createMock(ResponseInterface::class);
+        $response->expects(self::exactly(2))
```

Comment:
Not sure if there is a rule how to access phpunit's static helper methods within the symfony codebase, but in all other notifier tests they are accessed by `$this->...`